docs/ff-integrations/ads/admob.md

```diff
@@ -0,0 +1,481 @@
+---
```

Comment:
Please check the Arcades on the page. Seems to be wrong.

Reply:
Fixed!